### DISCUSSION TITLE
Pandapower uses internal transmission line instead of external connection for import?

### DISCUSSION THREAD

**Discussion by steffen-aalborg** [e2nIEE/pandapower#1491] [Q&A]
Dear PP team/users,

I set up a transmission network model of Northern Denmark based on data from the Danish TSO. The model it is set up with 4 external connections, from where I want the model to import the electricity that is not provided by the internal supply in the area.

My issue is, that the model seems to try to import from an internal line instead of the external connection (see the overloaded line in the picture below)
![image](https://user-images.githubusercontent.com/40762587/153014872-d051eddb-0633-463f-804a-c187c8e3675f.png)

If I disable the line that is overloaded, then the issue is resolved and it splits out the needed import on the 4 external connections (see picture below)
![image](https://user-images.githubusercontent.com/40762587/153017227-6686a785-f712-4585-a898-b976bfa70865.png)

It is not because the busses that are connected to the line has high loads. I tried removing the loads and the issue was the same. I even tried to connect the line to other busses, which had no impact, the model want to use that specific line for the needed import, no matter where I connect it to.

So, can anyone tell me why the model chooses that line over the other lines? Everything seems to work fine, when I do not have that line connected. 

Thanks in advance.

**Comment by rbolgaryn** [comment]
Hi @steffen-aalborg ,

I think it depends on where the ext_grid elements are.

It is hard to tell without investigating the grid in detail. I suggest that you try using the feature of distributed_slack (you can find a tutorial that describes how to use it). This way, you can specify the contributions of the import nodes exactly.
